### DISCUSSION TITLE
用markdown寫文章架構

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#340] [General]
# 標題一：Python
![image](https://user-images.githubusercontent.com/86505454/149280418-ae137ac0-c6ce-462d-9721-5017fb633359.png)

## 標題二：C(C語言原來是一枝彎彎像C的鉛筆)
![image](https://user-images.githubusercontent.com/86505454/149280763-ca8c2ed1-016f-4d9e-8866-fd557aaf27fb.png)

### 標題三：C++
![image](https://user-images.githubusercontent.com/86505454/149280628-f8831177-1c5f-49bb-9eaf-f933bb5fb63b.png)

#### 標題四：HTML
![image](https://user-images.githubusercontent.com/86505454/149281897-947d59c5-65e8-438e-baf7-dceb0ec04baa.png)

##### 標題五：CSS
![image](https://user-images.githubusercontent.com/86505454/149281010-25efd0b2-1c58-4286-990b-2f173321cddd.png)

###### 標題六：javascript
![image](https://user-images.githubusercontent.com/86505454/149281116-e924e850-ab98-4213-a211-87f9f923bf30.png)
以上是我找到 _六種程式_ 的圖片。如果找 _**錯**_ 了，就和我說吧！我對這些**不熟**。

**Comment by chupochun** [comment]
原來這些程式的LOGO是長這樣的

**Comment by JunTingLin** [comment]
標題1~6就是這樣使用
另外可以看這個喔!!!
[🖋Markdown神啊!(整理與分享) #377](https://github.com/NCU-GS4719-Python/Python-Community/discussions/377)

**Comment by pekoranoinu** [comment]
感謝分享，又知道了些有趣的小知識。

**Comment by boson13579** [comment]
原來是酷酷的冷知識分享，不過到底誰會知道C語言是一隻彎彎的鉛筆拉，感覺可以當資工系冷知識考同學(?

**Comment by alstinlin** [comment]
太酷啦，寫c語言那麼久，第一次看到代表他們的圖片，不過我覺得鉛筆的圖案蠻可愛的，為甚麼c++不繼續沿用呢?

**Comment by RogelioKG** [comment]
之前因為在 Notion 做各種語言的筆記，
每個語言都有套他們的 logo（小弟強迫症），
我認同的 C 的 logo 是長這樣。

<img src="https://user-images.githubusercontent.com/86526364/242049921-ff1e6ed0-cca0-4f38-bb79-26f3086b0825.png" alt="C" width="600" height="600">

總之不是鉛筆啦🥹，
C 應該沒有真正標誌性的 logo，
畢竟 C 發跡的年代在 1970，
那個年代大部分的語言似乎都找不太到 logo，
（那時候的編譯環境沒有圖形介面，
教學書也沒有酷酷的封面，
沒有 logo 也是情有可原）
例如 Pascal、Basic。

**Comment by RogelioKG** [comment]
是說 C++ 的 logo 比較常看到的應該是這個。

<img src="https://user-images.githubusercontent.com/86526364/242049932-a14e316f-4624-4c00-838f-00b9214e981a.png" alt="C++" width="600" height="600">

樓主給個 C++ logo 事實上是 C 和 C++ 合體的 logo。
我會知道是因為我 Notion 的 C 和 C++ 是筆記在一塊 page 的，然後我用的就是那個 logo，因為它很酷🤭。應該是參考了 Java Eclipse 的設計。

![eclipse icon 1 3778a4cbe978d8dfd73c091706e26aa6c57c65aa](https://github.com/NCU-GS4719-Python/Python-Community/assets/86526364/48bbd243-0dc1-43d7-8df9-d2760c44afaa)

**Comment by brian431** [comment]
Markdown語法相當地實用，若是能熟悉地掌握它將能加速做筆記的速度，做出來的筆記也會有較好的排版及一制性。

**Comment by KozakHou** [comment]
Markdown語法跟LaTex又有一些相似。
寫技術文件的時候學會兩種語法其實是很便利的。尤其是在數學表達式的時候。
